crates/client/l1-messages/src/error.rs

```diff
@@ -0,0 +1,43 @@
+use mp_felt::Felt252WrapperError;
+
+#[derive(thiserror::Error, Debug, PartialEq)]
+pub enum L1EventToTransactionError {
+    #[error("Failed to convert Calldata param from L1 Event: `{0}`")]
+    InvalidCalldata(Felt252WrapperError),
+    #[error("Failed to convert Contract Address from L1 Event: `{0}`")]
+    InvalidContractAddress(Felt252WrapperError),
+    #[error("Failed to convert Entrypoint Selector from L1 Event: `{0}`")]
+    InvalidEntryPointSelector(Felt252WrapperError),
+    #[error("Failed to convert Nonce param from L1 Event: `{0}`")]
+    InvalidNonce(Felt252WrapperError),
+}
+
+#[derive(thiserror::Error, Debug)]
+pub enum L1MessagesConfigError {
+    #[error("File with L1 Messages Worker config not found")]
+    FileNotFound(#[from] std::io::Error),
+    #[error("Failed to deserialize L1 Messages Worker Config from config file")]
```

Comment:
`"File with L1 Messages Worker config not found: {0}"`

we can use a bit more information on why it failed